src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+{
+	"global_executed_by": "Ausgeführt von {user_name}#{user_discriminator}",
+	"global_missing_permission": ":lock: Fehlende Rechte",
+	"global_missing_permission_bot": "Dem Bot fehlen Berechtigungen um dies durchzuführen! Bitte überprüfe die Rechte für den Bot.\n{missing}",
+	"global_user_not_found": "{emote_warn} Benutzer konnte nicht gefunden werden",
+	"global_channel_not_found": "{emote_warn} Channel konnte nicht gefunden werden",
+	"global_mod_action_log": "{action} by: {moderator_tag} ({moderator_id}) | Target: {target_tag} ({target_id}) | Grund: {reason}",
+	"global_logging_mod": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` für **{reason}** `[#{infractionId}]`",
+	"global_logging_unban_auto": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` Nachdem die Strafe abgelaufen ist `[#{infractionId}]`",
+	"global_logging_mod_temp": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` bis **{until}** für **{reason}** `[#{infractionId}]`",
+	"global_no_reason": "Keine Begründung gegeben",
+	"global_loading": "{emote_loading} Lädt...",
+	"global_not_in_voice": "{emote_warn} Benutzer ist mit keinem Voice-Channel verbunden!",
+	"global_unknown_error": "{emote_warn} Ein Unbekannter Fehler ist aufgetreteb, Bitte versuche es später wieder wenn der Fehler immer noch ist, kontaktiere bitte einen Entwickler",
+
+	"global_cannot_action_self": "{emote_warn} Du kannst diese Aktion nicht an dir Selbst durchführen!",
+	"global_cannot_action_owner": "{emote_warn} Du kannst diese Aktion nicht am Server-Besitzer durchführen!",
+	"global_cannot_action_role_equal": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da du keine Höhere Rolle als der User hast!",
+	"global_cannot_action_role_equal_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da meine Rolle niedriger ist als die Rolle vom User!",
+	"global_cannot_action_role_higher": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, Da er eine höhere Rolle als du hast!",
+	"global_cannot_action_role_higher_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da er eine Höhere Rolle als ich hat!",
+	"global_cannot_action_bot_self": "{emote_warn} You cannot perform this action on the bot!",
```

Untranslated. - On hold bc of 1st vs 3rd person.
```suggestion
	"global_cannot_action_bot_self": "{emote_warn} Du kannst diese Aktion nicht an mir durchführen!",
```

---

Thx for the Help